inductiva/fluids/scenarios/wind_tunnel/README.md

```diff
@@ -8,27 +8,37 @@ To initialize the scenario, the user can define the following parameters:
 
 Now, the user is ready to simulate the steady state. Here, the user chooses the object to be inserted inside the wind tunnel. This object is defined with a geometry file in STL or OBJ format. Notice that, the required meshing step will automatically be made before starting the simulation. The meshing is done with the [snappyHexMesh tool](https://www.openfoam.com/documentation/user-guide/4-mesh-generation-and-conversion/4.4-mesh-generation-with-the-snappyhexmesh-utility) of OpenFOAM.
 
+To test this scenario we have available one example of a vehicle geometry in OBJ format - [download it here](../../../../resources/geometry/test_vehicle.obj). The user can also use his own geometry file, as long as it is in STL or OBJ format.
+
 The simulation parameters available for the user to configure are:
 - `num_iterations`: Set the maximum number of iterations for the iterative algorithm to converge.
 - `resolution`: Controls the resolution of the meshing that is done prior to the simulation. The higher the resolution, the finer the meshing. Possibilities: "high", "medium", "low".
 
 Moreover, the hardware and interaction are configured with the usual general parameters - `machine_group`, `run_async`, `n_cores`.
 Launching a simulation returns a task object, which can be used to verify the status of the simulation, get the simulation outputs and access post-processing tools. See more in the [Tasks section](../../../../README.md).
 
+
+
+
 ### Example:
 
+To run this example download a vehicle geometry in OBJ format from [here](../../../../resources/geometry/test_vehicle.obj) and
+do not forget to insert your API Key (get one by filling this [form](https://docs.google.com/forms/d/e/1FAIpQLSflytIIwzaBE_ZzoRloVm3uTo1OQCH6Cqhw3bhFVnC61s7Wmw/viewform?usp=sf_link)).
+
 ```python
 import inductiva
 
+inductiva.api_key = "YOUR_API_KEY"
+
 # Initialize the scenario
 scenario = inductiva.fluids.scenarios.WindTunnel(
```

Comment:
Is this still this long? (just asking...)

Reply:
Yes! But I think we can remove scenarios for now and do `inductiva.fluids.WindTunnel()`, since the simulators are not anymore under fluids specifically.